napari_allencell_annotator/view/annotator_view.py

```diff
@@ -247,8 +261,17 @@ def get_curr_annots(self) -> List[Any]:
             a list of annotation values.
         """
         annots = []
-        for i in self.annot_list.items:
-            annots.append(i.get_value())
+        point_annots: dict[str, list[tuple[int]]] = self.viewer.get_all_point_annotations()
+
+        for item in self.annot_list.items:
+            if item.type == ItemType.POINT:
+                annot_name: str = item.name.text()
+                if annot_name in point_annots:
+                    annots.append(point_annots[annot_name])
+                else:
+                    annots.append(None)
+            else:
+                annots.append(item.get_value())
```

Comment:
reduce if else statements complexity. with comments

Reply:
modified in https://github.com/aics-int/napari-allencell-annotator/pull/94/commits/6b03d5ea958b6083dffdf0c057d3bbdca31f2de0